pkg/sql/schemachanger/scplan/testdata/alter_table

```diff
@@ -0,0 +1,559 @@
+create-table
+CREATE TABLE defaultdb.foo (i INT PRIMARY KEY)
+----
+
+ops
+ALTER TABLE defaultdb.foo ADD COLUMN j INT
+----
+Stage 0
+	*scop.MakeAddedColumnDeleteOnly
```

Comment:
I'm getting nitpicky now: it'd be nice if the yaml indentation levels matched the outer indentation levels.